crds/rancher-resource-set.yaml

```diff
@@ -29,29 +31,23 @@ resourceSelectors:
           values: ["true"]
   - apiVersion: "v1"
     kindsRegexp: "^serviceaccounts$"
-    namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-|^fleet-"
+    namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-|^fleet-|^cluster-fleet-"
     namespaces:
       - "rancher-operator-system"
   - apiVersion: "v1"
     kindsRegexp: "^configmaps$"
+    namespaceRegexp: "^fleet-|^cluster-fleet-"
     namespaces:
       - "cattle-system"
-      - "fleet-system"
   - apiVersion: "rbac.authorization.k8s.io/v1"
     kindsRegexp: "^roles$|^rolebindings$"
-    namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
-  - apiVersion: "rbac.authorization.k8s.io/v1"
```

Comment:
Why is rolebinding deleted?

Reply:
This is purposeful since we moved it, but the PR is difficult to read in its current state. I'll change it. https://github.com/rancher/backup-restore-operator/pull/74#issuecomment-788536396